### DISCUSSION TITLE
Storing config files in a shared location

### DISCUSSION THREAD

**Discussion by mikekazakov** [mikekazakov/nimble-commander#246] [Ideas]
Implement a decent support for storing settings in arbitrary locations (i.e. for Dropbox/Google Drive etc) and possibly for iCloud Drive.

**Comment by m-cerveny** [comment]
@mikekazakov 
My +1 for this one. Would this do an "import" of existing settings at first choosing of a location?

**Comment by mikekazakov** [comment]
Surely, when I'll come to.

**Comment by BashTux1** [comment]
+1 for being able to backup all your NC settings to a file.

**Comment by singalen** [comment]
`~/.config/nimblecommander` seems like a right place to save settings. 
Would also love to edit color settings (themes?) by hand in a human-readable format.
